src/styles/catalogue.scss

```diff
@@ -0,0 +1,44 @@
+.catalogue-box {
+  max-width: 1236px;
+  width: 100%;
+  min-height: 1084px;
+  height: fit-content;
+  text-align: center;
+  margin-top: 50px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  & h1 {
+    margin-bottom: 30px;
+  }
+}
+
+.products-box {
+  max-height: fit-content;
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(4, auto);
+  gap: 10px;
+  justify-content: center;
+  text-align: center;
+  margin: 0px;
+}
+
+@media (max-width: 1250px) {
+  .products-box {
+    grid-template-columns: repeat(3, auto);
+  }
+}
+
+@media (max-width: 960px) {
```

Comment:
I see a lot of media querys here for several screensizes, try to make it as simple as posible (it will save you future headaches), like having 2: tablet and desktop.

Also consider making custom and more readable mq mixins, like this:

``` scss
$breakpoint-tablet: 768px;
$breakpoint-desktop: 1024px;

@mixin tablet {
  @media (min-width: #{$breakpoint-tablet}) {
    @content;
  }
}

@mixin desktop {
  @media (min-width: #{$breakpoint-desktop}) {
    @content;
  }
}

.some-class {
  // Base styles

  @include tablet {
    // Tablet styles
  }

  @include desktop {
    // Desktop styles
  }
}
```